package/yast2-registration.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Tue Aug 14 13:10:04 UTC 2018 - knut.anderssen@suse.com
+
+- Modified addon selection filter for filtering all the testing
```

Comment:
NP: Use "all *development* versions" also here.

Reply:
My fault, I was already merging it